src/routing/approved.rs

```diff
@@ -200,18 +270,18 @@ impl Approved {
         Ok(commands)
     }
 
-    pub async fn handle_infrastructure_message(
+    pub async fn handle_networkinfo_msg(
         &mut self,
         sender: SocketAddr,
-        message: InfrastructureMessage,
+        message: NetworkInfoMsg,
     ) -> Vec<Command> {
         match message {
-            InfrastructureMessage::GetSectionRequest(name) => {
-                debug!("Received GetSectionRequest({}) from {}", name, sender);
+            NetworkInfoMsg::GetSectionQuery(name) => {
```

Comment:
just to double confirm that this renaming is agreed by others, right?

Reply:
Not more agreed than the first choice `InfrastructureMessage` was (not at all I think). Simply my view of what it should be,

Reply:
> you can have a map of `BTreeMap<SocketAddr, (EndUserPk, HashSet<SocketAddr>)>`
> and then impl the `get` function for EndUserRegistry` to avoid having two maps?

I think the two maps are needed since we use both socket_addr and socket_id to key into the maps (at different places).